conf/incomeSources.routes

```diff
@@ -283,12 +283,12 @@ GET          /agents/income-sources/manage/your-details-uk-property            c
 GET          /income-sources/manage/your-details-foreign-property              controllers.incomeSources.manage.ManageIncomeSourceDetailsController.show(isAgent: Boolean = false, incomeSourceType: IncomeSourceType = ForeignProperty, id: Option[String] = None)
 GET          /agents/income-sources/manage/your-details-foreign-property       controllers.incomeSources.manage.ManageIncomeSourceDetailsController.show(isAgent: Boolean = true, incomeSourceType: IncomeSourceType = ForeignProperty, id: Option[String] = None)
 
-GET          /income-sources/manage/business-will-report                        controllers.incomeSources.manage.ManageObligationsController.showSelfEmployment(changeTo: String, taxYear: String)
-GET          /agents/income-sources/manage/business-will-report                 controllers.incomeSources.manage.ManageObligationsController.showAgentSelfEmployment(changeTo: String, taxYear: String)
-GET          /income-sources/manage/uk-property-will-report                     controllers.incomeSources.manage.ManageObligationsController.showUKProperty(changeTo: String, taxYear: String)
-GET          /agents/income-sources/manage/uk-property-will-report              controllers.incomeSources.manage.ManageObligationsController.showAgentUKProperty(changeTo: String, taxYear: String)
-GET          /income-sources/manage/foreign-property-will-report                controllers.incomeSources.manage.ManageObligationsController.showForeignProperty(changeTo: String, taxYear: String)
-GET          /agents/income-sources/manage/foreign-property-will-report         controllers.incomeSources.manage.ManageObligationsController.showAgentForeignProperty(changeTo: String, taxYear: String)
+GET          /income-sources/manage/business-will-report                        controllers.incomeSources.manage.ManageObligationsController.show(changeTo: String, taxYear: String, incomeSourceType: IncomeSourceType = SelfEmployment)
```

Comment:
this change would imply changes in ATs required, and this is strange that current set of ATs not failing

Reply:
The url hasn't changed here and just the function in the controller is now just one shared controller rather than 3 separate ones.